packages/UI/src/components/molecules/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import Icon from '../../../assets/icons/rawSvg/check-circle.svg';
+import {
+  Wrapper,
+  Progress,
+  CheckIcon,
+  PercentageWrapper,
+  Percentage,
+} from './StyledProgressBar';
+import type { ProgressBarProps } from '.';
+
+const ProgressBar: React.FC<ProgressBarProps> = (props) => {
+  return (
+    <Wrapper percentage={props.percentage}>
+      <Progress percentage={props.percentage} />
+      <PercentageWrapper>
+        <Percentage percentage={props.percentage}>
+          {props.percentage}%
```

Comment:
I think this might be buggy if we pass in a value bigger than 100 or less than 0